src/app/psbt.ts

```diff
@@ -25,30 +26,26 @@ function isHexadecimal(str) {
     return str.length % 2 === 0 && hexRegex.test(str);
 }
 
-const getPsbt = (psbtContent) => {
-    const psbt = isHexadecimal(psbtContent)
-        ? bitcoin.Psbt.fromHex(psbtContent, {
-              network: NETWORK,
-          })
-        : bitcoin.Psbt.fromBase64(psbtContent, {
-              network: NETWORK,
-          });
-
-    return psbt;
-};
-
-const getPsbtBase64 = (psbtContent) => {
-    const buffer = Buffer.from(psbtContent, 'hex');
-    return buffer.toString('base64');
-};
-
-const Psbt = function (config) {
+const Psbt = function (config: Config) {
     const addressModule = Address(config);
     const cryptoModule = Crypto(config);
 
     const psbtModule = {
-        getPsbt,
-        getPsbtBase64,
+        getPsbt: (psbtContent) => {
+            const psbt = isHexadecimal(psbtContent)
+                ? bitcoin.Psbt.fromHex(psbtContent, {
+                      network: config.NETWORK,
```

Comment:
what's the difference between config.network and just network?

Reply:
The FE can override the config so there a case when could be different networks.

<img width="883" alt="image" src="https://github.com/deezy-inc/nosft-core/assets/126987350/ce741a49-a037-4b94-8a6f-a38b0fb78efd">